tap_mssql/__init__.py

```diff
@@ -100,13 +109,21 @@ def schema_for_column(c, config):
         result.maximum = 2 ** (bits - 1) - 1
 
     elif data_type in FLOAT_TYPES:
-        result.type = ["null", "number"]
-        result.multipleOf = 10 ** (0 - (c.numeric_scale or 17))
+        if use_singer_decimal:
+            result.type = ["null","string"]
+            result.format = "singer.decimal"
+        else:
+            result.type = ["null", "number"]
+            result.multipleOf = 10 ** (0 - (c.numeric_scale or 17))

```

Comment:
Singer decimal allows large numbers to be set to strings in transport so precision is not lost. The JSONSchema attribute `multipleOf` is dropped for singer decimals.